plugins/inventory/openstack.py

```diff
@@ -152,8 +156,7 @@ def main():
         elif args.host:
             inventory.get_host(args.host)
     except shade.OpenStackCloudException as e:
-        print(e.message)
-        sys.exit(1)
+        sys.exit(e.message)
```

Comment:
Pushed a new commit to fix.  Looking at the other changes in https://github.com/ansible/ansible/pull/10776, that PR is really inconsistent in this regard.